uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences.js

```diff
@@ -0,0 +1,64 @@
+var Promise = require('es6-promise').Promise;
```

Comment:
We are keeping everything related to leave and absence in a single folder, why make an exception here. I mean When I see a folder called leave and absence I would expect everything related to that inside.